cypress/integration/ete/register_email_sent.spec.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Is this file meant to be part of this PR?

Reply:
yep, mistake where I deleted it in the last upgrade Friday slipup 😳